x/examples/outline-connectivity-app/app_mobile/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
Do we need to put copyright header to HTML files like [this](https://github.com/Jigsaw-Code/outline-client/blob/master/src/www/index_electron.html#L1)?